jupyter-widgets-htmlmanager/src/htmlmanager.ts

```diff
@@ -45,11 +54,11 @@ class HTMLManager extends widgets.ManagerBase<HTMLElement> {
      */
     protected loadClass(className: string, moduleName: string, moduleVersion: string) {
         return new Promise(function(resolve, reject) {
-            if (moduleName === '@jupyter-widgets/controls') {
+            if (coreWidgetModules.indexOf(moduleName) >= 0) {
```

Comment:
The `@jupyter-widgets/base` base classes (like DOMWidgetView, WidgetModel, etc.) shouldn't be exported in `@jupyter-widgets/controls` - I think you'll need two checks here, one for each of your `coreWidgetModules`. See https://github.com/jupyter-widgets/ipywidgets/blob/36dd0aa54b810c60a58d9c18afee590f530dc2a8/widgetsnbextension/src/manager.js#L97